app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,879 @@
+
+const databases = new Databases(client);
+
+await databases.createDocument(
+    'marvel',
+    'movies',
+    ID.unique(),
+    {
+        title: 'Spiderman',
+        year: 2002,
+        reviews: [
+            'review1',
+            'review2'
+        ]
+    }
+)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Flutter</h3>
+        <div class="ide" data-lang="dart" data-lang-label="Flutter SDK">
+            <pre class="line-numbers"><code class="prism language-dart" data-prism>import 'package:appwrite/appwrite.dart';
+
+final client = Client()
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint    
+    .setProject('5df5acd0d48c2');               // Your project ID  
+
+final databases = Databases(client);
+
+await databases.createDocument(
+  databaseId: 'marvel',
+  collectionId: 'movies',
+  documentId: ID.unique(),
+  data: {
+    'title': 'Spiderman',
+    'year': 2002,
+    'reviews': [
+        'review1',
+        'review2'
+    ]
+  },
+);</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Apple</h3>
+        <div class="ide" data-lang="swift" data-lang-label="Apple SDK">
+            <pre class="line-numbers"><code class="prism language-swift" data-prism>import Appwrite
+
+let client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint        
+    .setProject("5df5acd0d48c2")                // Your project ID      
+
+let databases = Database(client: client)
+
+databases.createDocument(
+    databaseId: "marvel",
+    collectionId: "movies",
+    documentId: ID.unique(),
+    data: [
+        "title": "Spiderman",
+        "year": 2002,
+        "reviews": [
+            "review1",
+            "review2"
+        ]
+    ]
+)</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Android</h3>
+        <div class="ide" data-lang="kotlin" data-lang-label="Android SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import io.appwrite.Client
+import io.appwrite.services.Database
+import io.appwrite.ID
+
+val client = Client()
+    .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint    
+    .setProject("5df5acd0d48c2")                // Your project ID  
+
+val databases = Database(client)
+
+databases.createDocument(
+    databaseId = "marvel",
+    collectionId = "movies",
+    documentId = ID.unique(),
+    data = mapOf(
+        "title" to "Spiderman",
+        "year" to 2002,
+        "reviews" to listOf(
+            "review1",
+            "review2"
+        )
+    )
+)</code></pre>
+        </div>
+    </li>
+</ul>
+
+<h2><a href="#query" id="query">Querying</a></h2>
+
+<p>Querying is currently not available in the beta version of Appwrite Relationships but will be added in a later version.</p>
+
+<h2><a href="#update" id="update">Updating Relationships</a></h2>
+
+<p>Relationships can be updated by updating the relationship attribute.</p>
```

Comment:
Im not sure this is a good recommendation for the reason that the main point of relationships is to be able to update it from the related collection rather than through the parent collection.

The same result can be achieved by directly updating the related `movies` attribute in the `reviews` collection

Reply:
🤔 I'm not sure I understand what this means.

This example updates the relationship meaning you add, change, or remove which documents are related, not the document's contents.